designsafe/static/styles/ng-designsafe.css

```
@@ -979,4 +979,10 @@ i[class^="icon-ls-pre/post"]:before,
     }
     .curation-req-doi:before {
         content: "\e910";
+    }
+    .curation-recon:before {
+        content: "\e912";
+    }
+    .curation-download:before {
+        content: "\e913";
     }
```

Could you add a newline here?
```suggestion
    }

```